R/mod-panel-section.R

```diff
@@ -186,32 +188,32 @@ mod_panel_section_server <- function(input, output, session, synapse, syn, user,
   })
   certified <- check_certified_user(user$ownerId, syn = syn)
   
-  observeEvent(input$submission, {
-    if (!is.null(input$submission) && nchar(input$submission) > 0) {
```

Comment:
is removing this if statement going to make it do something weird if Refresh is clicked when no submission is selected?

Reply:
No it should be good! i put `req()` calls in for the dependency data so it should handle it properly. I just removed this because, one could in theory refresh the data without a submission being selected (though thinking about it, i'm not sure what the expected behavior of that is exactly. it would pull the new reviews table but you still wouldnt see anything until you click a submission anyway)